zebra/zebra_srv6_vty.c

```diff
@@ -354,8 +354,8 @@ static int zebra_sr_config(struct vty *vty)
 			inet_ntop(AF_INET6, &locator->prefix.prefix,
 				  str, sizeof(str));
 			vty_out(vty, "   locator %s\n", locator->name);
-			vty_out(vty, "    prefix %s/%u\n", str,
-				locator->prefix.prefixlen);
+			vty_out(vty, "    prefix %s/%u func-bits %u\n", str,
```

Comment:
In most cases, we don't display the _default_ value (if `func-bits` not set).

Reply:
Thanks for review.
I fixed to display only when the value is no-zero (which means `func-bits` is set).